packaging/setup/plugins/ovirt-engine-setup/ovirt-engine/network/ovirtproviderovn.py

```diff
@@ -404,10 +405,9 @@ def _getRSA():
 
         encrypted_password = _getRSA().public_key().encrypt(
             password.encode(),
-            # TODO replace PKCS1v15 with PSS if/when we know we do not
-            # need m2crypto compatibility. Would likely require changes
-            # also in the engine and in the ovn provider.
-            padding=padding.PKCS1v15(),
+            padding=padding.OAEP(
+                padding.MGF1(hashes.SHA256()), hashes.SHA256(), None
+            ),
         )
         return base64.b64encode(encrypted_password)
```

Comment:
so since this is the new padding it needs the extra "$" at the beginning too

Reply:
Right, fixed